src/main/resources/application.yaml

```diff
@@ -30,11 +67,63 @@ camunda:
       timeout: 10
 
 #----------------------------------------
-# Camunda integration settings
+# Integration settings
 #----------------------------------------
 # Rest client
 integration:
   camunda:
     connectTimeout: 5
     readTimeout: 20
-    url: ${config.camunda.base-url}
\ No newline at end of file
+    url: ${config.camunda.base-url}
+  casedata:
+    connectTimeout: 5
+    readTimeout: 20
+    url: ${config.casedata.base-url}
+  templating:
+    connectTimeout: 5
+    readTimeout: 20
+    url: ${config.templating.base-url}
+  messaging:
+    connectTimeout: 5
+    readTimeout: 20
+    url: ${config.messaging.base-url}
+
+#----------------------------------------
+# Circuit breakers
+#----------------------------------------
+resilience4j.circuitbreaker:
+  instances:
+    casedata:
+      ignoreExceptions: se.sundsvall.dept44.exception.ClientProblem
+    templating:
+      ignoreExceptions: se.sundsvall.dept44.exception.ClientProblem
+    messaging:
+      ignoreExceptions: se.sundsvall.dept44.exception.ClientProblem
+
+#----------------------------------------
+# MessagingMapper settings
+#----------------------------------------
+mappers:
+  messaging:
+    message: Ärendet avskrivs
+    filename: beslut.pdf
+    department: SBK(Gatuavdelningen, Trafiksektionen)
+    subject: Beslut från Sundsvalls kommun
+    htmlBody: <p><strong>Hej</strong></p><p>Du har f&aring;tt ett beslut fr&aring;n Sundsvalls kommun.</p><p>Med v&auml;nlig h&auml;lsning<br /><strong>Sundsvalls kommun</strong></p>
+    plainBody: |-
+      Hej
+      
+      Du har fått ett beslut från Sundsvalls kommun.
+
+      Med vänlig hälsning
+      Sundsvalls kommun
+    contact-info-email: sundsvalls.kommun@sundsvall.se
+    contact-info-phonenumber: 4660191000
```

Comment:
The international format for phone numbers are usually something like:
`+46 60 191000`

However, the text seems to be in swedish only and the content is probably targeted to local citizens? Maybe use the "local" format instead?
`060 191000`